deployment/k8s/base/daemonset.yaml

```diff
@@ -21,6 +21,8 @@ spec:
         securityContext:
           privileged: true
         volumeMounts:
+          # 如果configmap.yaml中指定了kubeletRootDir，则需要调整为相同的路径
+          # 如：kubeletRootDir=/data/kubelet，则mountPath=/data/kubelet/device-plugins
           - name: device-plugin
```

Comment:
just a note.

if kubelet root dir is different the two mounts below need to be updated in deployment. (thiis yaml is just example)

you dont want to mount entire */kubelet dir since it may cause issues as kubelet maintains other mounts under this root dir.

Reply:
also remove this comment :)